src/components/data/LocalVariablesContent.js

```diff
@@ -47,25 +48,60 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
     setEditIndex()
     setEditKey()
     setEditValue()
+    setEditDescription()
     setEditType()
   }
 
+  const DESCRIPTION_MAX_LENGTH = 200
+  const DESCRIPTION_SUFFIX = '__DESCRIPTION__'
+
+  function isDescriptionKey(k) {
+    return k.endsWith(DESCRIPTION_SUFFIX)
+  }
+
+  function toDescriptionKey(k) {
+    return k + DESCRIPTION_SUFFIX
+  }
+
+  function renameAttribute(kv) {
+    const [k, v] = kv
+    if (isDescriptionKey(k)) {
+      return {
+        key: k.substring(0, k.length - DESCRIPTION_SUFFIX.length),
+        description: v
+      }
+    }
+    return { key: k, value: v }
+  }
+
+  function toDisplayedAttribute(arr) {
+    const obj = Object.assign({}, ...arr)
+    return [obj.key, obj.value, obj.description]
+  }
```

Comment:
We use [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#comparing_traditional_functions_to_arrow_functions) instead of traditional functions. Could you please convert them for consistency?

Reply:
Nit: We typically use `verb`s when naming functions (e.g. `getDisplayedAttribute` instead of `toDisplayedAttribute`).

Reply:
Done